### ISSUE TITLE
Cluster Domain and Ingress Class ignored by install checks?

### ISSUE THREAD

**Issue by cablespaghetti** [acorn-io/runtime#1665]
I've tried both 0.6.0 and 0.7.0 alpha1 and have the same results.

Installing with:
`acorn install --cluster-domain sandbox.empennage.net --ingress-class-name ingress-nginx-internal --internal-registry-prefix nope.dkr.ecr.eu-west-1.amazonaws.com/acorn/`

(I also tried with a `.` in front of the cluster domain).

Results in the checks failing I assume due to an ingress with no class and an `acorn.io` domain being created for the check:
```
NAME                           CLASS    HOSTS                     ADDRESS   PORTS   AGE
acorn-check-ingress-nl8z4whc   <none>   inflight-check.acorn.io             80      3s
```

Using --checks=false (or `--skip-checks` for 0.7.0) sorts everything out. The `ConfigMap` seems to have the expected values.

By the way, the warning in 0.7.0 (I know it's alpha I'll let you off ;) ) still has the old removed --checks flag.

```
  •  WARNING:  Post-install checks failed. Use `acorn check` to debug or `acorn install --checks=false` to skip
                 IngressCapability: Ingress not ready (test timed out after 1 minute)
 ```

**Comment by cjellick** [comment]
Thanks for the report. Will take a look.

**Comment by iwilltry42** [comment]
Hi @cablespaghetti !
We're not actually accessing the ingress during the check but only wait for it to be populated with an address by the Ingress controller. That should work for a not-externally-routable address even.

Regarding the Ingress Class, which could actually break this check: Do you have more than one ingress class? If yes, do you have the one you want set as the default ingress class (annotation `ingressclass.kubernetes.io/is-default-class`)?

**Comment by cjellick** [comment]
bumping to v0.8

**Comment by iwilltry42** [comment]
@cablespaghetti thanks for the feedback. I figured there was a regression which should be fixed by #1730

**Comment by sangee2004** [comment]
@cablespaghetti This issue should be fixed and available in `acorn version v0.8.0-rc4+b6a95e8a`- https://github.com/acorn-io/runtime/releases/tag/v0.8.0-rc4.

**Comment by cjellick** [comment]
closing this out until we hear back otherwise.
